### PR TITLE
Landing page done

### PR DESCRIPTION
**Pull request by nikakhalatiani**
image

### CODE REVIEW

File: src/components/ui/button.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  extends React.ButtonHTMLAttributes<HTMLButtonElement>,
+    VariantProps<typeof buttonVariants> {
+  asChild?: boolean
+}
+
+const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
+  ({ className, variant, size, asChild = false, ...props }, ref) => {
+    const Comp = asChild ? Slot : "button"
+    return (
+      <Comp
```
Comment (dkurmaeva) [inline src/components/ui/button.tsx]: Please avoid using shortened names

**Comment by dkurmaeva** [comment]
Please add screenshot to PR description. It is helpful for review and if in future you need to return to the PR

**Comment by dkurmaeva** [comment]
I recommend to use ESLint and Prettier

**Review by dkurmaeva — CHANGES_REQUESTED** [review]

File: src/components/ui/button.tsx (hunk 1 above)
Comment (nikakhalatiani) [reply src/components/ui/button.tsx]: This is provided element from shadcn library, I can try to change it to adapt to naming convention you suggest

**Review by dkurmaeva — APPROVED** [review]

**Pull request merged**